packages/edit-post/src/components/keyboard-shortcut-help-modal/config.js

```diff
@@ -44,6 +44,10 @@ const globalShortcuts = {
 			description: __( 'Show or hide the settings sidebar.' ),
 			ariaLabel: shortcutAriaLabel.primaryShift( ',' ),
 		},
+		{
+			keyCombination: [ 'Esc', 'Esc' ],
```

Comment:
Turns out this keyboard command might be a bit too fast for some users. I noticed to get it to happen I had to press <kbd>Esc</kbd> twice in a row *pretty quickly*. I'm not if that's an OS-level setting (I'm guessing it's something our keyboard shortcut handler defines), but at its current speed it's *way* too tight a tolerance for users with mobility issues.

We should probably find another shortcut for this. I know there are so few, but we'll have to 🤷‍♂️

I'm open to suggestions and I'll try to come up with one.

Reply:
Possible keyboard options I've tried thus far:

* <del><kbd>⌥⌘B</kbd></del> (Used by Chrome for Bookmarks Bar)
* <del><kbd>⇧⌘B</kbd></del> (Used by Chrome for Bookmarks Page)